src/main/java/base/Figure.java

```diff
@@ -0,0 +1,19 @@
+package base;
+
+public class Figure implements AreaCalculable, Drawable {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, `Figure` should be an abstract class, because you are not supposed to create an instance of a 'Figure' itself, but rather only of its specific subclasses like `Circle`, `Square`, etc.